package.json

```diff
@@ -205,7 +206,8 @@
     "coveralls": "cat ./coverage/lcov.info | coveralls",
     "lint": "eslint '**/*.js'",
     "toc": "markdown-toc -i README.md",
-    "semantic-release": "semantic-release"
+    "semantic-release": "semantic-release",
+    "postinstall": "opencollective postinstall"
```

Comment:
can you remove this for now? I’m happy to add it later once we have some more progress to show that we achieved thanks to the money from the collective. 

Please remove the `opencollective` dependency from line 176, too